commands.go

```diff
@@ -11,10 +10,14 @@ import (
 	"github.com/pathfinder-cm/pathfinder-agent/agent"
 	"github.com/pathfinder-cm/pathfinder-agent/daemon"
 	"github.com/pathfinder-cm/pathfinder-go-client/pfclient"
+	"github.com/pathfinder-cm/pathfinder-agent/util"
 	log "github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
 )
 
+//"errors"
```

Comment:
please remove this unnecessary extra comments and linebreaks

Reply:
Done. f46ae53